src/config.ts

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable class-methods-use-this */
+export interface ConfigInterface {
+  ProjectID: string;
+  APISecret: string;
+  FrontendAPI: string;
+  BackendAPI: string;
+  ShortSessionCookieName: string;
+  JWTIssuer: string;
+  CacheMaxAge: number;
+}
+
+export const DefaultBackendAPI = 'https://backendapi.corbado.io';
+export const DefaultFrontendAPI = `https://${process.env.PROJECT_ID}.frontendapi.corbado.io`;
+export const DefaultShortSessionCookieName = 'cbo_short_session';
+export const DefaultCacheMaxAge = 60 * 1000; // 60 * 1000 = 60000 milliseconds, which is equivalent to 1 minute.
+
+class Configuration implements ConfigInterface {
```

Comment:
@sam-corbado Rename to "Config" please :) then in line with other SDKs

Reply:
Done.